File: 2012/June/ex15.py
#!/usr/bin/python
from sys import argv

#get script and input file names by unpack two strings
script, filename = argv

#open the input file
txt = open(filename)

#print the input file name and it's content.
print "Here's your file %r:" %filename
print txt.read()

txt.close()

#prompt to get another plain file.
print "Type the filename again:"
file_again = raw_input(">")

#open the file just input according to the file name.
txt_again = open(file_again)

#print out the content of the file.
print txt_again.read()
txt_again.close()

File: Learn_Ruby_The_Hard_Way/ex15.rb
#get filename from the first argument
filename = ARGV.first

prompt = "> "
#open the file
txt = File.open(filename)

puts "Here's your file: #{filename}"
#print the file content
puts txt.read()

puts "I'll also ask you to type it again:"
print prompt
#get the filename from STDIN
file_again = STDIN.gets.chomp()

#open the file whose name is from STDIN
txt_again = File.open(file_again)

#print the file content
puts txt_again.read()
